specutils/io/default_loaders/wcs_fits.py

```diff
@@ -27,25 +36,25 @@ def identify_wcs1d_fits(origin, *args, **kwargs):
     # Default FITS format is BINTABLE in 1st extension HDU, unless IMAGE is
     # indicated via naming pattern or (explicitly) selecting primary HDU.
     if origin == 'write':
-        return ((args[0].endswith(('wcs.fits', 'wcs1d.fits', 'wcs.fit')) or
-                 (args[0].endswith(('.fits', '.fit')) and whdu == 0)) and not
-                hasattr(args[2], 'uncertainty'))
+        return (args[0].endswith(('wcs.fits', 'wcs1d.fits', 'wcs.fit')) or
+                (args[0].endswith(('.fits', '.fit')) and whdu == 0))
 
     hdu = kwargs.get('hdu', 0)
     # Check if number of axes is one and dimension of WCS is one
     with read_fileobj_or_hdulist(*args, **kwargs) as hdulist:
         return (hdulist[hdu].header.get('WCSDIM', 1) == 1 and
                 (hdulist[hdu].header['NAXIS'] == 1 or
-                 hdulist[hdu].header.get('WCSAXES', 0) == 1 )and not
+                hdulist[hdu].header.get('WCSAXES', 0) == 1 or
+                hdulist[hdu].header.get('DISPAXIS', -1) >= 0 ) and not
```

Comment:
Indentation should match inner parenthesis.
```suggestion
                 hdulist[hdu].header.get('WCSAXES', 0) == 1 or
                 hdulist[hdu].header.get('DISPAXIS', -1) >= 0 ) and not
```